CGATPipelines/Pipeline/Execution.py

```diff
@@ -368,8 +368,8 @@ def run(**kwargs):
     options.update(kwargs.items())
 
     # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
+    options['cluster_options'] += " " + options.get('job_options',
+                                                    "")
```

Comment:
@sebastian-luna-valero: Thats fine for pipelines we write here, for use only here, but we need to be able to run other peoples pipelines and for other people to be able to run ours. 

@AndreasHeger: Yes, I think that separating out the cluster options from the job options is definitely the way to go.